monkey/infection_monkey/utils/timer.py

```diff
@@ -1,4 +1,5 @@
 import time
+from typing import Optional
 
 
 class Timer:
```

Comment:
Batching telemetry messenger should also use this?

Reply:
It does.
https://github.com/guardicore/monkey/pull/1856/commits/0be709a9eb8b9999247cc4e4c2a03d0372e2fae7